examples/Factorial/NumberEncoder.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Factorial
+{
+    using DotNetty.Buffers;
+    using DotNetty.Codecs;
+    using DotNetty.Transport.Channels;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Numerics;
+    using System.Text;
+    using System.Threading.Tasks;
+
+    public class NumberEncoder : MessageToMessageEncoder<BigInteger>
+    {
+        protected override void Encode(IChannelHandlerContext context, BigInteger message, List<object> output)
+        {
+            IByteBuffer buffer = context.Allocator.Buffer();
+
+            byte[] data = message.ToByteArray();
```

Comment:
>ToByteArray [](start = 34, length = 11)

in Java BigInteger.toByteArray() returns Big-Endian value which is in line with the default byte order in byte buffers. .NET's BigInteger.ToByteArray() returns Little-Endian ordered byte sequence. While not a problem for the sample, isn't good as a best practice. Maybe add a comment on Endianness not honoring ByteOrder of the buffer..